kitsune/sumo/jinja2/base.html

```diff
@@ -327,7 +327,9 @@ <h5 class="mzp-c-footer-heading">{{ _('Firefox Accounts') }}</h5>
   {% endif %}
 {% endif %}
 
-<script src="{{ url("wafflejs") }}"></script>
+{% if cached_page or waffle.switch("wafflejs") %}
```

Comment:
Enabling it for all cached pages won't benefit us much. We will end up nullifying this check for a big percentage of our traffic. Let's remove it completely and load it for everyone